### PR TITLE
UIU-679: Users App: Use documented react-intl patterns instead of stripes.intl

### PR DESCRIPTION
**Pull request by Godlevskyi**
Updated this repo to only use react-intl constructs outlined at 
https://github.com/folio-org/stripes-core/blob/master/doc/i18n.md

### CODE REVIEW

File: src/LoanActionsHistory.js [hunk 1]
```diff
@@ -432,11 +518,11 @@ class LoanActionsHistory extends React.Component {
               formatter={loanActionsFormatter}
               visibleColumns={['actionDate', 'action', 'dueDate', 'itemStatus', 'Source']}
               columnMapping={{
```
Comment (cherewaty) [inline src/LoanActionsHistory.js]: I think `columnMapping` right now has to receive strings - see https://github.com/folio-org/ui-data-import/pull/20 and https://issues.folio.org/browse/STCOM-399

Comment (Godlevskyi) [reply src/LoanActionsHistory.js]: Oh.. I see to calculate the width;

File: src/AccountActionsHistory.js [hunk 2]
```diff
@@ -237,58 +250,109 @@ class AccountActionsHistory extends React.Component {
 
           <Row>
             <Col xs={1.5}>
-              <KeyValue label={this.props.stripes.intl.formatMessage({ id: 'ui-users.details.field.feetype' })} value={_.get(account, ['feeFineType'], '-')} />
+              <KeyValue
+                label={<FormattedMessage id="ui-users.details.field.feetype" />}
+                value={_.get(account, ['feeFineType'], '-')}
```
Comment (aliaksei-chumakou) [inline src/AccountActionsHistory.js]: maybe it makes sense to import only `get` from `lodash/get`

Comment (VictorSoroka1) [reply src/AccountActionsHistory.js]: @aliaksei-chumakou I think it is not an issue anymore. @ttamj, could you please confirm that `wepback` plugin will be added to tree-shake `lodash` dependencies?

Comment (ttamj) [reply src/AccountActionsHistory.js]: @VictorSoroka1, yes that is correct.

File: src/settings/WaiveSettings.js [hunk 3]
```diff
@@ -17,21 +21,26 @@ class WaiveSettings extends React.Component {
...
     return (
       <this.connectedControlledVocab
-        stripes={this.props.stripes}
+        stripes={stripes}
         validate={(item, index, items) => validate(item, index, items, 'nameReason', label)}
         baseUrl="waives"
-        records="waiver"
-        label={this.props.stripes.intl.formatMessage({ id: 'ui-users.waives.label' })}
-        labelSingular={this.props.stripes.intl.formatMessage({ id: 'ui-users.waives.singular' })}
+        records="waives"
```
Comment (Yurii-Danylenko) [inline src/settings/WaiveSettings.js]: Please, replace `waives` with `waiver`

**Pull request merged**